src/main/java/roboy/context/ContextGUI.java

```diff
@@ -11,17 +9,17 @@
 import java.util.Map;
 
 /**
- * A simple GUI with the goal of showing the attribute values and histories in the Context.
+ * A simple GUI showing the values and histories in the Context with their content.
  */
 public class ContextGUI {
```

Comment:
My IDE tells me that this is never used. There is also no ``main(...)`` inside so I can start it. Could you add a comment how to use it please? I guess just call the ``run()`` function?